tools/fiptool/fiptool: Permission denied

**Describe the bug**

Observed in CI the following error:

```
make: execvp: tools/fiptool/fiptool: Permission denied
make: *** [Makefile:1453: /__w/zephyr/zephyr/twister-out/fvp_base_revc_2xaemv8a_smp_ns/samples/application_development/out_of_tree_driver/sample.drivers.out_of_tree/tfa/fvp/release/fip.bin] Error 127
make: Leaving directory '/__w/zephyr/modules/tee/tf-a/trusted-firmware-a'
ninja: build stopped: subcommand failed.
```

**To Reproduce**

```
west build -b fvp_base_revc_2xaemv8a_smp_ns samples/application_development/out_of_tree_driver -p
```

(not always reproducible, CI re-run may pass)

**Expected behavior**

CI build **always** succeeds.

**Impact**

CI failures

**Logs and console output**

See above.

**Environment (please complete the following information):**
 - OS: CI image
 - Commit SHA or Version used 147d9cb1272b39e119025953acdb3d19ba52e45e

**Additional context**

https://github.com/zephyrproject-rtos/zephyr/actions/runs/3225514752/jobs/5277945064

--- a/west.yml
+++ b/west.yml
@@ -235,7 +235,7 @@ manifest:
       groups:
         - tee
     - name: trusted-firmware-a
-      revision: d29cddecde614d81cbec1fb0086cdaebd77d3575
+      revision: 28f5e137837f1c1a7a7b2af2dd8bb778c0a27532
       path: modules/tee/tf-a/trusted-firmware-a
       groups:
         - tee

